### PR TITLE
Implement hasMonitor and tests

### PR DESCRIPTION
**Pull request by zackthehuman**
Addresses #38 for master.

### CODE REVIEW

File: src/runtime/index.js [hunk 1]
```diff
@@ -49,11 +49,15 @@ export default class Heimdall {
     this._session.events.push(OP_ANNOTATE, NULL_NUMBER, NULL_NUMBER, info);
   }
 
+  hasMonitor(name) {
+    return Boolean(this._monitors.has(name));
```
Comment (hjdivad) [inline src/runtime/index.js]: what's the reason for `Boolean` here?  seems fine to just return `this._monitors.has(name)` and rely on truthiness, no?

Comment (zackthehuman) [inline src/runtime/index.js]: Yeah, test failed because `null` !== `false`. Shall I loosen the test?

**Comment by stefanpenner** [comment]
@zackthehuman / @hjdivad whats blocking this?

**Comment by zackthehuman** [comment]
@stefanpenner Nothing but lack of time :) I had made this change a while back but forgot to rebase and sync my fork. Should be up-to-date now.

**Pull request merged**